### DISCUSSION TITLE
能添个talkatone的分流规则吗

### DISCUSSION THREAD

**Discussion by MMMinggg1** [blackmatrix7/ios_rule_script#1128] [General]
谢谢

**Comment by mottzz87** [comment]
talkatone我弄得比较简单粗暴，增加一个端口号5222的直连规则就可以

**Reply by mottzz87** [reply]
自建了个，拿去用吧
https://raw.githubusercontent.com/mottzz87/crules/main/rule/talkatone.list

**Reply by MMMinggg1** [reply]
> 自建了个，拿去用吧
> 
> https://raw.githubusercontent.com/mottzz87/crules/main/rule/talkatone.list

谢谢你
